### ISSUE TITLE
A11y > Investigate valid use of `aria-invalid` and `aria-required` for vertical tab

### ISSUE THREAD

**Issue by Blackbaud-SteveBrush** [blackbaud/skyux-tabs#85]
I don't think we're using them correctly. See:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-invalid_attribute

**Source code:**
https://github.com/blackbaud/skyux-tabs/blob/master/src/app/public/modules/vertical-tabset/vertical-tab.component.ts#L39-L43

**Ported pull request (from skyux2):**
https://github.com/blackbaud/skyux2/pull/1798

**Comment by Blackbaud-SteveBrush** [comment]
Closing issue due to inactivity. If you believe this should still be addressed, please file a new issue and include any additional context and justification. Thank you.
